pkg/ride/generate/main.go

```diff
@@ -488,13 +488,21 @@ func functionsV5() map[string]string {
 	m["423"] = "stringToBigInt"
 	m["424"] = "stringToBigIntOpt"
 	m["425"] = "medianListBigInt"
+	m["1009"] = "hashScriptAtAddress"
 	m["1020"] = "invoke"
+	m["1054"] = "isDataStorageUntouched"
+	m["1055"] = "intFromSelfState"
+	m["1056"] = "booleanFromSelfState"
+	m["1057"] = "bytesFromSelfState"
+	m["1058"] = "stringFromSelfState"
 	m["1081"] = "calculateLeaseID"
 	m["1092"] = "simplifiedLease"
 	m["1093"] = "fullLease"
 	m["LeaseCancel"] = "leaseCancel"
-	m["1054"] = "isDataStorageUntouched"
-	m["1009"] = "hashScriptAtAddress"
+	m["@extrNative(1055)"] = "intValueFromSelfState"
```

Comment:
why do we need both "intValueFromSelfState" and "intFromSelfState"? Don't they do literally the same?

Reply:
Because the first one will return `throw` on `Unit` value, but the latter won't.